server/testdriver/core/src/main/java/org/infinispan/server/test/core/TestSystemPropertyNames.java

```diff
@@ -33,4 +33,10 @@
     * The path to a local installation of the server
     */
    public static final String INFINISPAN_TEST_SERVER_DIR = "org.infinispan.test.server.dir";
+   /**
+    * The log file for the container driver server
+    * It should be the file name located in the module server/tests/src/test/resources/configuration
```

Comment:
If we add a `builder..copy` call to the driver, it will be possible to provide any log file from the local filesystem as it will be copied into the container.

Reply:
Should it be the full path for the local filesystem ?

Reply:
It should handle relative and absolute paths I would say.

Reply:
If the file is absolute we copy it to the container otherwise we use the log file provided in the `server/conf` folder.

I have explained https://github.com/infinispan/infinispan/pull/8309/files#diff-cd50636dc19e28e7810727179207e57fR119-R120 why we can't use `builder.copy`